src/content/docs/browser/new-relic-browser/getting-started/TEST-PAGE-Browser.mdx

```diff
@@ -1,41 +0,0 @@
----
```

Comment:
This is the page that @mmfred added. Let's keep this file around.

Reply:
you read my mind. was going to hold on merging this until i can sync drupal prod to dev and run migration again as part of this PR

Reply:
Sweet! Thank you!!